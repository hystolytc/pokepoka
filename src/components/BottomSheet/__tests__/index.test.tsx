import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import { BottomSheet } from '../index'

const describeComponent = '<BottomSheet /> test case'

describe(describeComponent, () => {
  const { container } = render(
    <BottomSheet isOpen={true}>
      <p>BottomSheet Test</p>
    </BottomSheet> 
  )

  test(`Render ${describeComponent} correctly`, () => {
    expect(container).toMatchSnapshot()
  })
})