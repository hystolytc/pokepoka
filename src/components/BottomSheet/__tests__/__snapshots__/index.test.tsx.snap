// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BottomSheet /> test case Render <BottomSheet /> test case correctly 1`] = `
<div>
  <div
    class="css-1b64myx"
  >
    <div>
      <p>
        BottomSheet Test
      </p>
    </div>
  </div>
</div>
`;
